server/liveblog/syndication/syndication.py

```diff
@@ -94,6 +97,12 @@ def send_syndication_post(self, post, action='created'):
             logger.warning('Not sending post "{}": syndicated content.'.format(post['_id']))
             return
 
+        items = extract_post_items_data(post)
```

Comment:
I think that it would make sense to put exclusion logic in a separate method, like _post_for_syndication, because it will grow